config.json

```diff
@@ -229,137 +229,152 @@
         "slug": "hello-world",
         "name": "Hello World",
         "uuid": "f77dc7e3-35a8-4300-a7c8-2c1765e9644d",
-        "practices": [],
+        "practices": [
+          "basics"
+        ],
         "prerequisites": [],
-        "difficulty": 1,
-        "topics": [
-          "strings"
-        ]
+        "difficulty": 1
       },
       {
         "slug": "two-fer",
         "name": "Two Fer",
         "uuid": "74515d45-565b-4be2-96c4-77e58efa9257",
-        "practices": [],
-        "prerequisites": [],
-        "difficulty": 1,
-        "topics": [
-          "conditionals-if",
-          "strings"
-        ]
+        "practices": [
+          "strings",
+          "conditionals-if"
+        ],
+        "prerequisites": [
+          "basics",
```

Comment:
Do not include basics if you include anything else (because any other concept will have basics as a prerequisite).

Reply:
I was in doubt if I should include basics or not, but ended up inserting it. Thanks for the suggestion, I've removed it from all exercises that have other prerequisites. It really makes more sense, otherwise all the exercises of the track would end up with basics as a prerequisite.